torch/csrc/jit/passes/quantization.cpp

```diff
@@ -878,9 +890,6 @@ void InsertQuantDeQuantHelper::run(
   for (Value* v : input_values) {
     collectObserverNodesAndValueToQuantize(module, v);
   }
-  GRAPH_DUMP("Before Remove Observers:", graph);
-  removeObservers(module, graph.get());
-  GRAPH_DUMP("Before Quantize Tensors:", graph);
   Value* self = graph->inputs()[0];
   quantizeTensors(module, graph.get(), self);
   GRAPH_DUMP("After Quantize Tensors:", graph);
```

Comment:
Why can't we just move them after this line? That should do the trick and we won't need another (module-level) function for removing observers. It feels strange to keep observer nodes around in the graph where we already inserted all q-dq nodes.

Reply:
we have to keep the observer node around since we need them in order to get the quantization parameters, if they are removed we have to find another way to get these parameters and we'll create two path to get these parameters, that's probably not a great trade off.